pkgs/development/libraries/tdlib/default.nix

```diff
@@ -11,8 +11,8 @@ stdenv.mkDerivation {
     # The tdlib authors do not set tags for minor versions, but
```

Comment:
Let me know if I understood.

You basically keep track from `master` but do not use the `<version>-unstable-<date>` format because the upstream sets a version in CMakeLists?

Reply:
You basically need to find the commit with the version bump and that is the version.